packages/runtime/src/components/builtin/Text/EditableText.tsx

```diff
@@ -69,6 +69,11 @@ const EditableText = forwardRef(function EditableText(
   useImperativeHandle(
     ref,
     () => ({
+      getDomNode() {
+        const el = editor?.findDOMNode([])
```

Comment:
I noticed this morning that findDOMNode is deprecated. Does that worry you at all? 

https://beta.reactjs.org/reference/react-dom/findDOMNode

Reply:
Nope! I've talked to Sebastian Markbåge about this and basically the reason this hasn't been fully removed from React is because there's no alternative to it right now.

Ideally we can do this:

```tsx
<Fragment ref={domNodeRef}>
  <SomeUknownComponent />
</Fragment>
```

But that's not possible right now. I think that by the time they remove `findDOMNode`, then something equivalent to that will be possible and we'll have a suitable replacement.

Reply:
Cool. Good to know!